crates/sui-indexer-alt-jsonrpc/src/data/bigtable_reader.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::data::read_error::ReadError;
+use sui_kvstore::BigTableClient;
```

Comment:
This should be renamed to `BigtableClient` some other day since "Bigtable" is one word.